handlers/meta/whatsapp/templates.go

```diff
@@ -52,9 +52,9 @@ func GetTemplatePayload(templating MsgTemplating) *Template {
 		if strings.HasPrefix(k, "button.") {
 
 			for _, p := range v {
-				if strings.HasPrefix(p.Value, "http") {
+				if p.Type == "url" || strings.HasPrefix(p.Value, "http") {
 					component := &Component{Type: "button", Index: strings.TrimPrefix(k, "button."), SubType: "quick_reply"}
-					component.Params = append(component.Params, &Param{Type: "url", Text: p.Value})
+					component.Params = append(component.Params, &Param{Type: "text", Text: p.Value})
```

Comment:
https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages#button-parameter-object

type supported are payload and text